### PR TITLE
feat: add stylelint module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
Add https://github.com/nuxt-community/stylelint-module

### CODE REVIEW

**Comment by manniL** [comment]
Conflicts are present.

**Comment by ricardogobbosouza** [comment]
Resolve conflicts

File: template/_stylelint.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  rules: {
+
```
Comment (clarkdo) [inline template/_stylelint.config.js]: If there is no default config is needed here, can we add some descriptions and doc link here ?

Comment (ricardogobbosouza) [reply template/_stylelint.config.js]: Yes I agree.
Is there any expected stylelint configuration for nuxt like `@nuxtjs/styleint-config`?

Comment (clarkdo) [reply template/_stylelint.config.js]: I think default config is OK, maybe just adding desc and links for detailed config.

Comment (ricardogobbosouza) [reply template/_stylelint.config.js]: @clarkdo I added a little description
```js
// add your custom config here
// https://stylelint.io/user-guide/configuration
```

**Comment by ricardogobbosouza** [comment]
I'll add

**Comment by clarkdo** [comment]
Thanks!

**Comment by shoyuf** [comment]
When i use  @nuxtjs/stylelint-module above stylelint, my application doesn't build success and not emit any errors

**Comment by ricardogobbosouza** [comment]
Hi @shoyuf 
Fixed `@nuxtjs/stylelint-module` on `v3.1.1`

**Pull request merged**